### PR TITLE
fixed SQL Server widget

### PR DESCRIPTION
**Pull request by mlozbin-cybervisiontech**
WIKI - https://wiki.cask.co/display/CE/CDC+Solution+Enhancements

In scope of this PR:
- fixed Goldengate plugin widget (reference page)

### CODE REVIEW

**Comment by mlozbin-cybervisiontech** [comment]
@jayfjin, could you, please, take a look.

File: docs/CDCDatabase-streamingsource.md [hunk 1]
```diff
@@ -10,21 +10,23 @@ CDC source produces messages in CDC format.
...
-Ex: mysqlserver.net or 12.123.12.123.
+**Kafka Broker**: Kafka broker specified in host:port form. For example, example.com:9092.
 
-**Port**: SQL Server Port.
+**Kafka Topic**: Name of the topic to which Golden Gate publishes the DDL and DML changes.
 
-**Username**:  Username to use to connect to the specified database. Required for databases that need authentication. 
-Optional for databases that do not require authentication.
+**Default Initial Offset**: The default initial offset to read from. An offset of -2 means the smallest offset. 
+An offset of -1 means the latest offset. Defaults to -1. Offsets are inclusive.
```
Comment (jayfjin) [inline docs/CDCDatabase-streamingsource.md]: what does smallest or latest offset mean?

Comment (mlozbin-cybervisiontech) [reply docs/CDCDatabase-streamingsource.md]: smallest means from the beginning. Latest means from recent available topic offset.

Comment (jayfjin) [reply docs/CDCDatabase-streamingsource.md]: can we add this in the text?

Comment (mlozbin-cybervisiontech) [reply docs/CDCDatabase-streamingsource.md]: but current terminology corresponds to Kafka documentation https://kafka.apache.org/documentation/ (auto.offset.reset property for example)

Comment (jayfjin) [reply docs/CDCDatabase-streamingsource.md]: It's still not clear for someone unfamiliar with the terminology

Comment (mlozbin-cybervisiontech) [reply docs/CDCDatabase-streamingsource.md]: fixed

**Comment by mlozbin-cybervisiontech** [comment]
@jayfjin addressed all comments

**Pull request merged**